cara/tests/apps/calculator/test_report_json.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+import tornado.testing
+
+import cara.apps.calculator
+
+_TIMEOUT = 40.
+
+
+class TestCalculatorJsonResponse(tornado.testing.AsyncHTTPTestCase):
+    def setUp(self):
+        super().setUp()
+        self.http_client.defaults['request_timeout'] = _TIMEOUT
+
+    def get_app(self):
+        return cara.apps.calculator.make_app(xsrf_cookies=False)
+
+    @tornado.testing.gen_test(timeout=_TIMEOUT)
+    def test_json_response(self):
+        response = yield self.http_client.fetch(
+            request=self.get_url("/calculator/report-json"),
+            method="POST",
+            headers={'content-type': 'application/json'},
+            body=json.dumps(HTTP_POST_BODY)
+        )
+        self.assertEqual(response.code, 200)
+        self.assertTrue('prob_inf' in json.loads(response.body).keys())
+
+
+HTTP_POST_BODY = {
```

Comment:
Can you re-use the body from the ``baseline_form_data`` fixture, rather than having an explicit duplication of the input here?